.github/workflows/monthly-check.yml

```
@@ -0,0 +1,40 @@
+name: Monthly check
```

```suggestion
name: Monthly check GitHub usernames
```
For me, the name "Monthly check" is too generic and can be misleading.